### PR TITLE
[Docs] Add description of concept documents

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes https://github.com/exercism/v3/issues/2592

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
Requesting review from @exercism/reviewers

**Review by coriolinus — APPROVED** [review]

**Review by ekingery — APPROVED** [review]

File: docs/maintainers/generic-how-to-implement-a-concept-exercise.md [hunk 1]
```diff
@@ -15,8 +15,9 @@ Please also watch the following video:
 As this document is generic, the following placeholders are used:
 
 - `<LANG>`: the name of the track in kebab-case (e.g. `ruby`).
-- `<SLUG>`: the name of the exercise in kebab-case (e.g. `anonymous-methods`).
+- `<SLUG>`: the slug of the exercise in kebab-case (e.g. `calculator-conundrum`).
 - `<UUID>`: the exercise's unique UUID.
+- `<CONCEPT_SLUG>`: the slug of one of the exercise's concepts in kebab-case (e.g. `anonymous-methods`).
```
[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply docs/maintainers/generic-how-to-implement-a-concept-exercise.md:20]: This is the concept slug, not the exercise's slug, so in this case `anonymous-methods` makes sense.

**Review by verdammelt — APPROVED** [review]
Look OK to me.

File: docs/concept-exercises.md [hunk 2]
```diff
@@ -252,6 +234,43 @@ For more information, watch [this video][video-tests-file] and check [this examp
 
 For more information, watch [this video][video-example-file] and check [this example file][example-file].
 
+## Concept files
+
+For each concept that an exercise unlocks (which are the concepts listed in the exercise's `concepts` key in the track's `config.json` file), two concept documents must be added. The concept files for a concept are located in the track's `concepts/<concept>` directory.
+
+### `about.md`
+
+**Purpose:** Provide information about the concept(s) for a student to learn from.
```
Comment (iHiD) [inline docs/concept-exercises.md:243]:
```suggestion
**Purpose:** Provide information about the concept(s) for a student who has completed the concept execise to learn from and refer back to.
```

File: docs/maintainers/generic-how-to-implement-a-concept-exercise.md [hunk 3]
```diff
@@ -115,6 +115,18 @@ An entry should be added to the track's `config.json` file for the new concept e
 
 For more information, please read [this in-depth description][config.json] and check [this example file][example-config.json].
 
+## Add `concepts/<CONCEPT>/about.md` for each unlocked concept
```
Comment (iHiD) [inline docs/maintainers/generic-how-to-implement-a-concept-exercise.md:118]:
```suggestion
## Add `concepts/<CONCEPT>/about.md` for each taught concept
```

File: docs/maintainers/generic-how-to-implement-a-concept-exercise.md [hunk 4]
```diff
@@ -115,6 +115,18 @@ An entry should be added to the track's `config.json` file for the new concept e
...
+
+**Purpose:** Provide information about the concept(s) for a student to learn from.
+
+For more information, check [this example file][example-concept-about.md].
+
+## Add `concepts/<CONCEPT>/links.json` for each unlocked concept
```
Comment (iHiD) [inline docs/maintainers/generic-how-to-implement-a-concept-exercise.md:124]:
```suggestion
## Add `concepts/<CONCEPT>/links.json` for each taught concept
```

File: docs/maintainers/generic-how-to-implement-a-concept-exercise.md [hunk 5]
```diff
@@ -25,20 +26,25 @@ Any concept exercise in any v3 track requires at least the following files to be
...
                 |   ├── instructions.md
                 |   ├── hints.md
-                |   ├── after.md
                 |   └── source.md (required if there are third-party sources)
                 └── .meta
                     ├── design.md
                     └── config.json
 </pre>
 
+Note that the concept files should be created for each concept the exercise unlocks.
```
[the remark this answers is not in the document]
Comment (iHiD) [reply docs/maintainers/generic-how-to-implement-a-concept-exercise.md:46]:
```suggestion
Note that the concept files should be created for each concept the exercise teaches.
```

File: docs/concept-exercises.md [hunk 6]
```diff
@@ -252,6 +234,43 @@ For more information, watch [this video][video-tests-file] and check [this examp
 
 For more information, watch [this video][video-example-file] and check [this example file][example-file].
 
+## Concept files
+
+For each concept that an exercise unlocks (which are the concepts listed in the exercise's `concepts` key in the track's `config.json` file), two concept documents must be added. The concept files for a concept are located in the track's `concepts/<concept>` directory.
```
Comment (iHiD) [inline docs/concept-exercises.md:239]:
```suggestion
For each concept that an exercise teaches (which are the concepts listed in the exercise's `concepts` key in the track's `config.json` file), two concept documents must be added. The concept files for a concept are located in the track's `concepts/<concept>` directory.
```

**Review by SaschaMann — APPROVED** [review]

**Pull request merged**